frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -62,32 +64,25 @@ function FileManagerHistory:fetchStatuses(count)
 end
 
 function FileManagerHistory:updateItemTable()
-    -- try to stay on current page
-    local select_number = nil
```

Comment:
We no longer try ? :) Why not? Or is it magically ensured somewhere else?

Reply:
We pass -1 in `switchItemTable` and stay on the page even after removing an item.
Have I missed anything?

Reply:
No, I just didn't know (or forgot) about the effect of `-1`:
https://github.com/koreader/koreader/blob/962477e7c679c2724c369536d2a1e941c678ad1d/frontend/ui/widget/menu.lua#L1143-L1157
So, if it works as stated, no problem :)